snooty/test_postprocess.py

```diff
@@ -668,6 +668,29 @@ def test_language_selector() -> None:
             TabMustBeDirective
         ]
 
+    # Ensure we gracefully handle invalid tabs
+    with make_test(
```

Comment:
Since we don't do any validation with `TabMustBeDirective` in `postprocess.py`, this test feels a bit superfluous.

Reply:
difference for this test case is using `.. tabs::` instead of `.. tabs-drivers::` 😄 

I'll differentiate the two test cases in the comments 👍